sbt/src/sbt-test/plugins/dotty-sandwich/build.sbt

```diff
@@ -0,0 +1,24 @@
+ThisBuild / scalaVersion := "0.23.0"
+
+ThisBuild / resolvers += "scala-integration" at "https://scala-ci.typesafe.com/artifactory/scala-integration/"
+// TODO use 2.13.4 when it's out
+lazy val scala213 = "2.13.4-bin-aeee8f0"
+
+lazy val root = (project in file("."))
+  .aggregate(fooApp, fooCore, barApp, barCore)
+
+lazy val fooApp = (project in file("foo-app"))
+  .dependsOn(fooCore)
+
+lazy val fooCore = (project in file("foo-core"))
+  .settings(
+    scalaVersion := scala213
```

Comment:
For completeness sake, I think it'd be nice to have one or more tests involving crossScalaVersion and running ++compile, e.g. what happens if I have a scala 3 app that depends on a lib that cross-compiles between 2 and 3, and vice-versa.

Reply:
That's probably not gonna work.

Reply:
Am I misreading previous @smarter comments? A 2.13.4+ (not released yet) should be able to depend on a dotty library, so why wouldn't this visa-versa test work?

Reply:
`+` command is ignorant about the sandwich, so if there are a list of scalaVersions it wouldn't know how to align them. I'd much rather handle that scenario using sbt-projectmatrix - https://github.com/sbt/sbt-projectmatrix/issues/28

Reply:
thanks for explaining!